### ISSUE TITLE
name a was shadowed

### ISSUE THREAD

**Issue by srghma** [purescript/purescript-lists#160]
```
Warning found:
in module Data.List.Lazy.Types
at .psc-package/psc-0.12.3/lists/v5.4.0/src/Data/List/Lazy/Types.purs:199:12 - 199:13 (line 199, column 12 - line 199, column 13)

  Name a was shadowed.


See https://github.com/purescript/documentation/blob/master/errors/ShadowedName.md for more information,
or to contribute content related to this warning.

```

**Comment by natefaubion** [comment]
It looks like you might be using a 0.12.3 package set with 0.13. This warning was fixed as part of the 5.4.1 releas of purescript-lists.

**Comment by srghma** [comment]
tnx for quick reply)
